packages/next/src/utils.ts

```diff
@@ -39,9 +39,13 @@ import { getNextjsEdgeFunctionSource } from './edge-function-source/get-edge-fun
 import type { LambdaOptionsWithFiles } from '@vercel/build-utils/dist/lambda';
 import { stringifySourceMap } from './sourcemapped';
 import type { RawSourceMap } from 'source-map';
+import bytes from 'bytes';
 
 type stringMap = { [key: string]: string };
 
+const _prettyBytes = (n: number) => bytes(n, { unitSeparator: ' ' });
+export { _prettyBytes as prettyBytes }
```

Comment:
This could be simplified by exporting a named function which has the added benefit of showing the function name in the unlikely event of a stack trace. This file also contains several other exported functions that match the following style:

```suggestion
export function prettyBytes(n: number) {
  return bytes(n, { unitSeparator: ' ' });
}
```